client/src/components/LoginForm/SignupContent.jsx

```diff
@@ -48,6 +49,19 @@ class SignupContent extends Component {
     );
   }
 
+  async getYears() {
+    try {
+      let url = 'curriculums/getCurriculumData';
+      let r = await Api().get(url);
```

Comment:
looks good, just wondering, I think front end team is using axios, try looking into that. Not 100% sure if the same package can be used to complete what you're trying to do.